src/routes/devices.ts

```diff
@@ -185,19 +184,24 @@ export const receiveOnlineDependentDevices: RequestHandler = async (
 				online_dependent_devices,
 				devices.map(({ local_id }) => local_id),
 			);
-			await Bluebird.map(toBeProvisioned, (localId) =>
-				// Provision new dependent devices
-				resinApiTx.post({
-					resource: 'device',
-					body: {
-						uuid: randomstring.generate({ length: 62, charset: 'hex' }),
-						belongs_to__user: user,
-						belongs_to__application: dependent_app,
-						device_type: dependent_device_type,
-						local_id: localId,
-						logs_channel: randomstring.generate({ length: 62, charset: 'hex' }),
-					},
-					options: { returnResource: false },
+			await Promise.all(
+				toBeProvisioned.map(async (localId) => {
+					// Provision new dependent devices
+					await resinApiTx.post({
```

Comment:
Do we really benefit that much by having the stacktrace point to the map in here?
I mean, it sounds like an extra thing to remember to do while coding new things.

Reply:
It's probably not that big of a deal to miss it but the better the stack trace the easier a lot of things are to debug, so I leant towards too much rather than too little